docs-gb/installation/README.md

```diff
@@ -0,0 +1,67 @@
+
+## Local Environment
+Ideal for development and testing purposes, a local setup allows for quick iteration and experimentation with minimal overhead. Common tools include:
+* Docker Compose:
+Simplifies deployment by orchestrating Seldon Core components and dependencies in Docker containers. Suitable for environments without Kubernetes, providing a lightweight alternative.
+* Kind (Kubernetes IN Docker):
+Runs a Kubernetes cluster inside Docker, offering a realistic testing environment.
+Ideal for experimenting with Kubernetes-native features.
+
+## Production Environment
+Designed for high-availability and scalable deployments, a production setup ensures security, reliability, and resource efficiency. Typical tools and setups include:
+* Managed Kubernetes Clusters:
+Platforms like GKE (Google Kubernetes Engine), EKS (Amazon Elastic Kubernetes Service), and AKS (Azure Kubernetes Service) provide managed Kubernetes solutions.
+Suitable for enterprises requiring scalability and cloud integration.
+* On-Premises Kubernetes Clusters:
+For organizations with strict compliance or data sovereignty requirements.
+Can be deployed on platforms like OpenShift or custom Kubernetes setups.
+
+By selecting the appropriate installation mode—whether it's Docker Compose for simplicity, Kind for local Kubernetes experimentation, or production-grade Kubernetes for scalability—you can effectively leverage Seldon Core 2 to meet your specific needs.
+
+## Helm Charts
+
+| **Name of the Helm Chart**              | **Description**                                                                                                                                              |
+|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `seldon-core-v2-crds`      | Cluster-wide installation of custom resources.                                                                                                              |
+| `seldon-core-v2-setup`     | Installation of the manager to manage resources in the namespace or cluster-wide. This also installs default `SeldonConfig` and `ServerConfig` resources, allowing *Runtimes* and *Servers* to be installed on demand. |
+| `seldon-core-v2-runtime`   | Installs a `SeldonRuntime` custom resource that creates the core components in a namespace.                                                                 |
+| `seldon-core-v2-servers`   | Installs `Server` custom resources providing example core servers to load models.                                                                            |
+| `seldon-core-v2-certs`     | A default set of certificates for TLS.                                                                                                                       |
```

Comment:
these certs are provided as example as opposed to recommended production install and it also requires `cert-manager`, reword accordingly.

We also do not publish this chart in https://github.com/SeldonIO/helm-charts/tree/master/helm-charts

Reply:
deleted that row after checking  https://github.com/SeldonIO/helm-charts/tree/master/helm-charts